master/buildbot/test/unit/test_changes_bitbucket.py

```diff
@@ -261,6 +261,9 @@ class TestBitbucketPullrequestPoller(changesource.ChangeSourceMixin,
                                      TestReactorMixin,
                                      unittest.TestCase):
 
+    skip = ('BitbucketPullrequestPoller is broken. '
+            'See https://github.com/buildbot/buildbot/issues/4153')
```

Comment:
I'd rather fix the unit tests and the code. Disabling the test unconditionally is the best way to have bitbucket broken for all pythons